### PR TITLE
Remove Dashboard/Home button from the navbar

### LINKED ISSUE #16699: Proposal: Remove the Dashboard/Home button from the navigation bar
I am proposing to remove the Dashboard/Home button to reduce the amount of items in the navbar and have a simpler experience for users. The Gitea icon already points to the same location, and for accessibility, an aria-label with `Dashboard` or `Home` can be added to the link. Users will also be familiar with this navigation style as it used on other software forges.

### PR DESCRIPTION
**Pull request by n194**
This PR removes the Dashboard/Home buttons and adds its text to the title of the logo.

logged-in
logged-out

Closes #16699

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Comment by n194** [comment]
Would it make sense to remove the `Explore` button as well if the landing page is `explore` or `organizations`?

**Comment by delvh** [comment]
I don't see why the landing page would be `Explore`. 
I think that can stay in.

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
And I would like to add help back even if login.

**Review by tomaswarynyca — APPROVED** [review]

File: templates/base/head_navbar.tmpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="ui container" id="navbar">
 	<div class="item brand" style="justify-content: space-between;">
-		<a href="{{AppSubUrl}}/">
+		<a href="{{AppSubUrl}}/" title="{{if .IsSigned}}{{.i18n.Tr "dashboard"}}{{else}}{{.i18n.Tr "home"}}{{end}}">
```
Comment (axifive) [inline templates/base/head_navbar.tmpl]:
```suggestion
		<a href="{{AppSubUrl}}/" class="item poping up" data-content="{{if .IsSigned}}{{.i18n.Tr "dashboard"}}{{else}}{{.i18n.Tr "home"}}{{end}}" data-variation="tiny inverted">
```

Comment (axifive) [reply templates/base/head_navbar.tmpl]: I think we should stick to the same style

Screen Shot 2021-09-19 at 23 22 53
Screen Shot 2021-09-19 at 23 21 54

File: templates/base/head_navbar.tmpl [hunk 2]
```diff
@@ -23,12 +22,7 @@
 		{{end}}
 		<a class="item {{if .PageIsExplore}}active{{end}}" href="{{AppSubUrl}}/explore/repos">{{.i18n.Tr "explore"}}</a>
 	{{else if .IsLandingPageHome}}
```
Comment (axifive) [inline templates/base/head_navbar.tmpl]:
```suggestion
	{{else}}
```

**Review by axifive — CHANGES_REQUESTED** [review]

**Comment by axifive** [comment]
@lunny if user login help already exists in menu
Screen Shot 2021-09-20 at 03 14 07

File: templates/base/head_navbar.tmpl [hunk 3]
```diff
@@ -22,13 +21,8 @@
 		{{if .ShowMilestonesDashboardPage}}<a class="item {{if .PageIsMilestonesDashboard}}active{{end}}" href="{{AppSubUrl}}/milestones">{{.i18n.Tr "milestones"}}</a>{{end}}
 		{{end}}
 		<a class="item {{if .PageIsExplore}}active{{end}}" href="{{AppSubUrl}}/explore/repos">{{.i18n.Tr "explore"}}</a>
-	{{else if .IsLandingPageHome}}
```
Comment (lunny) [inline templates/base/head_navbar.tmpl:25]: Why we need to change these parts?

Comment (n194) [reply templates/base/head_navbar.tmpl:25]: @axifive suggested it and it made sense to generalize it.

**Review by wxiaoguang — APPROVED** [review]

File: templates/base/head_navbar.tmpl [hunk 4]
```diff
@@ -22,13 +21,8 @@
...
-		<a class="item {{if .PageIsHome}}active{{end}}" href="{{AppSubUrl}}/">{{.i18n.Tr "home"}}</a>
+	{{else}}
 		<a class="item {{if .PageIsExplore}}active{{end}}" href="{{AppSubUrl}}/explore/repos">{{.i18n.Tr "explore"}}</a>
```
Comment (axifive) [inline templates/base/head_navbar.tmpl]: As commented [above](https://github.com/go-gitea/gitea/pull/16844#discussion_r719656760) this solution is also good in my opinion
```suggestion
	{{else if .IsLandingPageOrganizations}}
		<a class="item {{if .PageIsExplore}}active{{end}}" href="{{AppSubUrl}}/explore/organizations">{{.i18n.Tr "explore"}}
	{{else}}
		<a class="item {{if .PageIsExplore}}active{{end}}" href="{{AppSubUrl}}/explore/repos">{{.i18n.Tr "explore"}}</a>
```

File: templates/base/head_navbar.tmpl (hunk 3 above)
Comment (axifive) [reply templates/base/head_navbar.tmpl:25]: > @axifive suggested it and it made sense to generalize it.

I just suggested bring your solution to work correctly. Maybe need leave a separate link for the organization
https://github.com/go-gitea/gitea/pull/16844/files#r719655772

File: templates/base/head_navbar.tmpl (hunk 4 above)
Comment (n194) [reply templates/base/head_navbar.tmpl]: Fixed in 711a339

**Comment by lunny** [comment]
CI failed is related.

**Comment by n194** [comment]
@lunny thanks, fixed.

**Review by 6543 — APPROVED** [review]

**Pull request merged**